bootstrap/mocks/things.go

```diff
@@ -70,19 +70,21 @@ func (svc *mainfluxThings) ViewThing(_ context.Context, owner, id string) (thing
 	return things.Thing{}, things.ErrNotFound
 }
 
-func (svc *mainfluxThings) Connect(_ context.Context, owner, chID string, thIDs ...string) error {
+func (svc *mainfluxThings) Connect(_ context.Context, owner string, chIDs []string, thIDs []string) error {
```

Comment:
Can you please use `chsID` and `thsID`

Reply:
That would read as channels ID which sounds like multiple channels sharing one ID.  This reads as channel IDs which sounds like multiple channel IDs, which is what this variable contains.  I'll change it if you want but I think it's misleading to what this variable contains.  Same goes for the following comments.

Reply:
@nwneisen I agree with you

Reply:
`channel IDs` sounds like multiple IDs of one channel. I would say that `channels IDs` is the best 😄. As you prefer guys, I let you vote.

Reply:
Agreed with @nwneinsen also. Keeping current implementation is more accurate IMHO.

Reply:
@nwneisen people voted. Approving this one!